src/js/api/config.js

```diff
@@ -105,6 +109,33 @@ define([
             config.skin = config.skin.replace('.xml', '');
         }
 
+        if (config.playbackRateControls) {
+            var playbackRateOptions;
+
+            var validRatesFilter = (value) => { return _.isNumber(value) && value >= 0.25 && value <= 4; };
+
+            if (_.isBoolean(config.playbackRateControls)) {
+                playbackRateOptions = [0.5, 1, 1.25, 1.5, 2];
+            } else if (_.isArray(config.playbackRateControls)) {
+                var filteredResults = config.playbackRateControls.filter(validRatesFilter);
+                playbackRateOptions = filteredResults.length ? filteredResults : false;
+            }
+
+            if (playbackRateOptions) {
```

Comment:
`playbackRateOptions` should be defined if `config.playbackRateControls` is set, so you don't need these conditions. I also don't understand why you'd reassign `config.playbackRateControls` AND set `config.playbackRateOptions`.